### DISCUSSION TITLE
Wyze Doorbell 2 - can be hacked?

### DISCUSSION THREAD

**Discussion by teredactle** [gtxaspec/wz_mini_hacks#649] [Q&A]
Team, is it possible to (easily) put this firmware on the Wyze Doorbell 2??
https://www.wyze.com/products/wyze-video-doorbell-v2

I'd love to get one of these to put on my side door, not really for use as a doorbell but more so to keep an eye on my kids while they play on the swings, climbing towers, etc...

**Comment by gtxaspec** [comment]
If you have one, open it and send photos.  Need to know what platform it's using.

update:
https://github.com/gtxaspec/wz_mini_hacks/issues/656
